test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -216,6 +242,10 @@ try
         {
             remove-item -force -recurse "${outputBaseFolder}/src"
         }
+        if ( Test-Path "$outputBaseFolder/assests" )
+        {
+            remove-item -force -recurse "${outputBaseFolder}/assets"
```

Comment:
Since you're modifying this, might as well fix the casing `Remove-Item`

Reply:
And casing parameters too.

Reply:
Fixed.